guides/architecture.rst

```diff
@@ -0,0 +1,18 @@
+.. _flux-architecture:
+
+#################
+Flux Architecture
```

Comment:
Is "architecture" the correct term here? This seems to be describing a high level overview of Flux Framework and many of the current components that are currently project under that umbrella. It could just be me, but when I see "Flux Architecture", I would expect to see something more along the lines of what's described [here](https://en.wikipedia.org/wiki/Software_architecture)

Maybe this document should be called "Flux Framework Overview and Components"?